### DISCUSSION TITLE
Is hangover supposed to be the official way to run wine or supported

### DISCUSSION THREAD

**Discussion by m1m1k4tz** [ptitSeb/box64#1475] [Q&A] [answered]
Should hangover be the official way for using wine I’ve read it uses box64 in a specific way so it’s not emulating a whole wine environment which might make things simpler/faster

**Comment by ptitSeb** [comment, marked as answer]
Hangover only handle 32bits x86 for now. When 64bits x86_64 is added (or handled by wine directly?), and integrate box64, I'll mention it in the readme.

**Reply by m1m1k4tz** [reply]
Sweet because it’s in progress right now https://github.com/AndreRH/hangover/issues/142
